packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -183,15 +184,10 @@ type ContextMenuRootContentPrimitive = Polymorphic.ForwardRefComponent<
 >;
 
 const ContextMenuRootContent = React.forwardRef((props, forwardedRef) => {
-  const { disableOutsidePointerEvents = true, ...contentProps } = props;
-  const context = useContextMenuContext(CONTENT_NAME);
   return (
     <MenuPrimitive.Content
-      {...contentProps}
+      {...props}
       ref={forwardedRef}
-      disableOutsidePointerEvents={context.open ? disableOutsidePointerEvents : false}
```

Comment:
It wasn't clear to me what purpose this `open` check was serving?

Reply:
Historically we've done that because `closed !== unmounted` when animating out. So, this will have enabled pointer events outside while the animation was playing out.